platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/metrics/EventIntakeMetrics.java

```diff
@@ -69,35 +70,35 @@ public class EventIntakeMetrics implements StaleEventObserver {
             new RunningAverageMetric.Config(INTERNAL_CATEGORY, "shouldCreateEvent").withFormat(FORMAT_10_1);
     private final RunningAverageMetric shouldCreateEvent;
 
-    private static final SpeedometerMetric.Config STALE_EVENTS_PER_SECOND_CONFIG = new SpeedometerMetric.Config(
-                    INTERNAL_CATEGORY, "staleEventsPerSecond")
-            .withDescription("number of stale events per second");
-    private final SpeedometerMetric staleEventsPerSecond;
+    private static final LongGauge.Config STALE_EVENTS_PER_SECOND_CONFIG = new LongGauge.Config(
+                    INTERNAL_CATEGORY, "staleEvents")
+            .withDescription("number of stale events since the node booted up");
+    private final LongGauge staleEventCount;
 
-    private static final SpeedometerMetric.Config STALE_SELF_EVENTS_PER_SECOND_CONFIG = new SpeedometerMetric.Config(
-                    INTERNAL_CATEGORY, "staleSelfEventsPerSecond")
-            .withDescription("number of stale self events per second");
-    private final SpeedometerMetric staleSelfEventsPerSecond;
+    private static final LongGauge.Config STALE_SELF_EVENTS_PER_SECOND_CONFIG = new LongGauge.Config(
```

Comment:
Your config variable names still say `PER_SECOND`, but these aren't per second anymore

Reply:
To me it seems that these metrics are per second counters as opposed to "since forever"

Reply:
> Your config variable names still say PER_SECOND, but these aren't per second anymore

Doh! Fixed.

> To me it seems that these metrics are per second counters as opposed to "since forever"

Kelly actually requested I switch these away from being per second. She said that we are planning on deprecating the speedometer metric because it's inefficient, and that we can configure grafana to display a "since forever" metric as just diffs, making it look pretty similar to what a speedometer would show. This makes sense to me... in general I think it's a good idea to do any data processing after the fact and not directly on the node.

Reply:
Ok, after further discussion, these are now accumulators. This means they are cleared out every time they are read, so it will be easy to interpret.